worker/transfer.go

```diff
@@ -478,10 +472,10 @@ func migrateSlab(ctx context.Context, s *object.Slab, hosts []sectorStore, locke
 	// move slow hosts to the back of the array
 	slow := make(map[types.PublicKey]int)
 	for _, h := range slowHosts {
-		slow[hosts[h].PublicKey()]++
+		slow[v3Hosts[h].HostKey()]++
 	}
-	sort.SliceStable(hosts, func(i, j int) bool {
-		return slow[hosts[i].PublicKey()] < slow[hosts[j].PublicKey()]
+	sort.SliceStable(v2Hosts, func(i, j int) bool {
```

Comment:
having a hard time finding the spot so I'm leaving this here.

In some spots it might make sense to also filter out / move back hosts with empty accounts.